src/frontend/src/icp/derived/icrc.derived.ts

```diff
@@ -105,6 +107,17 @@ export const sortedIcrcTokens: Readable<IcrcCustomToken[]> = derived(
 	([$icrcTokens]) => $icrcTokens.sort(sortIcTokens)
 );
 
+/**
+ * The list of Icrc tokens that are default for Chain Fusion, in the order provided by the static list.
+ */
+export const icrcChainFusionDefaultTokens: Readable<IcrcCustomToken[]> = derived(
+	[icrcTokens],
+	([$icrcTokens]) =>
+		ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS.map((canisterId) =>
```

Comment:
Curiosity question, is this implementation more performant than direcly filtering `$icrcTokens`?

```
$icrcTokens.filter(ledgerCanisterId => ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS.includes(ledgerCanisterId))
```

Really just a curiosity question.

Reply:
The only reason I didn't do that was to respect the order of `ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS`, since I was not sure if it is the same of `icrcTokens`. For sure it would be more performant your way, but it does not guarantee the order given by `ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS`

Reply:
all good this one @peterpeterparker ? does it make sense?